rust-semantics/preprocessing/constants.md

```diff
@@ -10,7 +10,11 @@ module RUST-CONSTANTS
 
     rule
         (const Name:Identifier : T:Type = V:Value;):ConstantItem:KItem
-        => setConstant(Name, implicitCast(V, T))
+        => setConstant(Name, implicitCast(V, T)) 
```

Comment:
Why is there a diff for this line?

Reply:
I'm honestly not sure. I think it may be a change of tabs for spaces, perhaps. Will try to figure it out.